client/flutter/flutter-netlify-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+FLUTTER_BRANCH=`grep channel: .metadata | sed 's/  channel: //g'`
+FLUTTER_REVISION=`grep revision: .metadata | sed 's/  revision: //g'`
+
+git clone https://github.com/flutter/flutter.git
+cd flutter
+git checkout $FLUTTER_BRANCH
+git pull origin $FLUTTER_BRANCH
+git checkout $FLUTTER_REVISION
+cd ..
```

Comment:
No need to clone everything:
```suggestion
mkdir flutter
cd flutter
git init
git remote add origin https://github.com/flutter/flutter.git
git fetch --depth 1 origin $FLUTTER_REVISION
cd ..
```

Reply:
ok I tried this and it seemed to fail pulling what was needed, so I've reverted